### ISSUE TITLE
Atom crashing upon "copy"

### ISSUE THREAD

**Issue by TheChymera** [atom/atom#6296]
Hi, I recently upgraded to `atom-0.184.0` and the required `atom-shell-0.22.3`. All works remarkably well, considering how severe the one thing that does *not* is:

Almost always (though strangely enough not really always) atom crashes whenever I try to copy something into its window. Not only that, but even if I have saved a file in a ~5s time window before the crash, it will undo whatever save that was :-/ This is not as far as I can tell contingent on what it is that I copy or where it is copied from.

This is seriously annoying! And sadly I can't seem to debug it , atom automatically detaches itself from the terminal so that I do not see any output...

**Comment by kevinsawicki** [comment]
Can you try upgrading to `0.190.0` which came out today?

**Comment by kevinsawicki** [comment]
This sounds very similar to https://github.com/atom/atom/issues/6221

**Comment by kevinsawicki** [comment]
Closing as a duplicate of #6221 please upgrade to 0.190 for the fix
